kuma/sample_apps.go

```diff
@@ -0,0 +1,29 @@
+package kuma
+
+import (
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/status"
+	"github.com/layer5io/meshkit/utils"
+)
+
+func (kuma *Kuma) installSampleApp(del bool, templates []adapter.Template) (string, error) {
+	st := status.Installing
+
+	if del {
+		st = status.Removing
+	}
+
+	for _, template := range templates {
+		contents, err := utils.ReadRemoteFile(string(template))
+		if err != nil {
+			return st, ErrSampleApp(err)
+		}
+
+		err = kuma.applyManifest([]byte(contents))
+		if err != nil {
+			return st, ErrSampleApp(err)
```

Comment:
Do we have an issue tracking the removal of “sample”? It good that we track attributes of applications. Perhaps, “sample” is beat tracked as an annotation in manifest?

Reply:
U mean change the name "sample" to something else? @leecalcote